spec/helper/functional-test.helper.js

```diff
@@ -543,17 +407,26 @@ class FunctionalTest {
             options.body = new Buffer(0);
         }
 
-        queryString = options.url.split("?");
-        path = queryString.shift();
-        queryString = queryString.join("?");
+        path = options.url;
+        queryString = querystring.stringify(options.query);
+
+        if (options.parameters) {
+            Object.keys(options.parameters).forEach((key) => {
```

Comment:
I believe the functional route tester handles this. I don't think it needs to be done in OpenToken.